go-controller/pkg/node/ovn_db_mgmt.go

```diff
@@ -0,0 +1,139 @@
+package node
+
+import (
+	"regexp"
+	"strings"
+	"time"
+
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/klog"
+
+	"github.com/ovn-org/ovn-kubernetes/go-controller/pkg/config"
+	"github.com/ovn-org/ovn-kubernetes/go-controller/pkg/kube"
+	"github.com/ovn-org/ovn-kubernetes/go-controller/pkg/metrics"
+	"github.com/ovn-org/ovn-kubernetes/go-controller/pkg/util"
+)
+
+func (n *OvnNode) ensureOvnRaft(db string) {
+	ticker := time.NewTicker(60 * time.Second)
+	klog.Infof("Starting ensure routine for Raft db: %s", db)
+	kclient := n.Kube.(*kube.Kube)
+	for {
+		select {
+		case <-ticker.C:
+			if dbOnNode, _ := metrics.CheckPodRunsOnGivenNode(kclient.KClient.(*kubernetes.Clientset),
+				"name=ovnkube-db", n.name, false); dbOnNode {
+				klog.Infof("Node %s has a OVN DB pod, ensuring OVN Raft membership", n.name)
+				n.ensureLocalRaftServerID(db)
+				n.ensureClusterRaftMembership(db)
```

Comment:
so every node Pod running ovnkube-raft nodes would do ensureClusterRaftMembership()? Is it better to only run it on the cluster leader?

Reply:
previously i was only running it on the ovn-kube master leader, but yeah I think it would make sense to only run it where DB leader currently resides.